internal/cmd/exodus.go

```diff
@@ -137,24 +137,29 @@ func exodusMain(ctx context.Context, cfg conf.Config, args args.Config) int {
 
 	uploadCount := 0
 	existingCount := 0
+	duplicateCount := 0
 
 	err = gwClient.EnsureUploaded(ctx, items,
 		func(uploadedItem walk.SyncItem) error {
 			uploadCount++
 			return nil
 		},
-		func(existingItem walk.SyncItem) error {
+		func(uploadedItem walk.SyncItem) error {
```

Comment:
Minor - while I don't think it affects the behavior of the code, changing the argument name here to uploadedItem seems odd.